src/main/resources/i18n/strings.properties

```diff
@@ -14,6 +14,8 @@ generic.button.copied=Copied!
 generic.button.done=Done
 generic.button.next=Next
 generic.button.print=Print
+## Selection
+generic.choicebox.autoSelection=Automatic
```

Comment:
Make it more specific, something like `preferences.volume.type.automatic`.

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2639/commits/b4c384acc9e23e0f387f0b400f3a50c7e26162dd